File: swift_/baekjoon/step4/8958.swift
import Foundation

let N = Int(readLine()!)!

for _ in 1...N{
    var score = 0
    var before = 0
    let quiz = readLine()!
    for i in quiz{
        if i == "O"{
            before += 1
            score += before
        }
        else{
            before = 0
        }
    }
    print(score)
}

File: python/with_java/solved.ac/class 1/8958.py
import sys
input = sys.stdin.readline

n = int(input())

for _ in range(n):
    cmd = input()
    score = 0
    plus = 1
    for c in cmd:
        if c == 'O':
            score += plus
            plus += 1
        else:
            plus = 1
    print(score)